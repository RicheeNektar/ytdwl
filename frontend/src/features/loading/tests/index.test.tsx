import TestRenderer from 'react-test-renderer';
import Loading from '..';

const content = 'Lorem Ipsum Text';

describe('Loading', () => {
  it('renders content', () => {
    const component = TestRenderer.create(
      <Loading>
        <p>{content}</p>
      </Loading>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

  it('renders loading circle', () => {
    const component = TestRenderer.create(
      <Loading isLoading>
        <p>{content}</p>
      </Loading>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
