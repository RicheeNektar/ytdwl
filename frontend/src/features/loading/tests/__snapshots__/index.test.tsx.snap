// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders content 1`] = `
<div>
  <p>
    Lorem Ipsum Text
  </p>
</div>
`;

exports[`Loading renders loading circle 1`] = `
<div>
  <div
    className="sc-bdnxRM hbdCeh"
  />
</div>
`;
